Automattic/woocommerce-payments
# Fix failing E2E pipelines

Fixes https://github.com/Automattic/woocommerce-payments/issues/8051
Fixes https://github.com/Automattic/woocommerce-payments/issues/7966

#### Changes proposed in this Pull Request
This PR
* Fixes tests that are failing on `develop` due to new notices selectors coming from WooCommerce
* Adds some backward compatibility for old selectors to be supported as well and thus makes [`WC 7.7.0` flow](https://github.com/Automattic/woocommerce-payments/actions/runs/7586821939/job/20665971286) pass as well
* Cleans up some unused code (more https://github.com/Automattic/woocommerce-payments/pull/8048#discussion_r1459306170

> All tests pass now https://github.com/Automattic/woocommerce-payments/actions/runs/7586821939










#### Testing instructions





* run https://github.com/Automattic/woocommerce-payments/actions/workflows/e2e-test.yml against this branch once more and confirm all tests pass

-------------------


**Post merge**
